### DISCUSSION TITLE
注册失败

### DISCUSSION THREAD

**Discussion by DY13208** [walinejs/waline#622] [Q&A] [answered]
一直注册失败是啥意思啊？

**Comment by lizheming** [comment]
提供下地址我帮你看看是什么原因

**Reply by DY13208** [reply]
https://bing-api-ebon.vercel.app/

**Reply by DY13208** [reply]
是不是我域名解析的有问题？

**Comment by DY13208** [comment, marked as answer]
![屏幕截图 2021-10-29 113814](https://user-images.githubusercontent.com/70312842/139371885-449adb47-bec3-4d9e-bcdd-38589be73f11.png)

**Reply by lizheming** [reply]
LEAN_SERVER 是指的 leancloud 的 server 地址，不是填你的评论 server 地址。另外如果你是用的 LeanCloud 国际版的话，是不需要配 LEAN_SERVER 的。这个在文档里有说的~

**Comment by deeper-coder** [comment]
请问怎么解决啊，我也存在第一次登录无法注册的情况qaq，网址https://waline-server-beryl.vercel.app/

**Comment by DY13208** [comment]
看上面那个评论你变量弄错了应该

**Comment by deeper-coder** [comment]
好的我看看




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月10日(星期四) 晚上8:39
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [walinejs/waline] 注册失败 (Discussion #622)





 
看上面那个评论你变量弄错了应该

**Reply by DY13208** [reply]
对了我当初是因为域名的解析出问题填错了你可以试试看你的是否出错

**Comment by deeper-coder** [comment]
检查了一下发现变量都填写正确，所以主要想问一下：1. LeanCloud只需要开一个项目然后把三个key帮到vercel上吗，不需要绑定我个人网站域名嘛？
2. 注册时的个人网站那一栏填写的是什么？
谢谢你的解答~




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月10日(星期四) 晚上8:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [walinejs/waline] 注册失败 (Discussion #622)





 
对了我当初是因为域名的解析出问题填错了你可以试试看你的是否出错

**Reply by DY13208** [reply]
Leancloud 是要绑定域名的你可以看官方这个教程https://waline.js.org/guide/get-started.html#leancloud-%E8%AE%BE%E7%BD%AE-%E6%95%B0%E6%8D%AE%E5%BA%93

**Reply by lizheming** [reply]
1. 国际版不需要绑定域名，只需要相关的 KEY
2. 注册的个人网站地址随便填即可，它是评论时候头像链接地址

**Comment by deeper-coder** [comment]
好的谢谢~我再试试



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Austin ***@***.***&gt;; 
发送时间: 2022年3月10日(星期四) 晚上9:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [walinejs/waline] 注册失败 (Discussion #622)





  
国际版不需要绑定域名，只需要相关的 KEY
 
注册的个人网站地址随便填即可，它是评论时候头像链接地址

**Comment by Ginkgovio** [comment]
我用的是国外得，变量也是对的，但是注册一直500错误

---
I use foreign products, and the variables are correct, but the registration has been 500 errors

**Reply by SinzMise** [reply]
你有没有配置RECAPTCHA_V3_KEY和RECAPTCHA_V3_SECRET变量？（我之前配置这两个变量也出现了500）

---
Have you configured the RECAPTCHA_V3_KEY and RECAPTCHA_V3_SECRET variables? (500 also appeared when I configured these two variables before)
